packages/lib-classifier/src/components/Classifier/components/Banners/components/Banner/Banner.js

```diff
@@ -23,6 +23,20 @@ function Label () {
   )
 }
 
+export function Tooltip ({ mode, tooltipText }) {
```

Comment:
should extract `Tooltip` (and related tests) to own file in `./components`?

Reply:
Actually, we should replace it with the `Tooltip` that is part of the component library. This was written prior to that one I think.

Reply:
This can be done in a separate PR.